Admin/BaseMediaAdmin.php

```diff
@@ -106,7 +106,17 @@ public function getNewInstance()
 
         if ($this->hasRequest()) {
             if ($this->getRequest()->isMethod('POST')) {
-                $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
+                $uniqid = $this->getUniqid();
+
+                if (method_exists('Symfony\Component\HttpFoundation\JsonResponse', 'transformJsonError')) {
+                    // NEXT_MAJOR remove this block when dropping sf < 2.8 compatibility
+                    $media->setProviderName(
+                        $this->getRequest()->get(sprintf('%s[providerName]', $uniqid), null, true)
+                    );
+                } else {
+                    $providerParams = $this->getRequest()->get($uniqid);
+                    $media->setProviderName($providerParams['providerName']);
+                }
```

Comment:
Not sure you did the right thing here… If you got a merge conflict here, that might be b/c someone (probably me) cleaned this in a previous commit on master. Let me check.